assets/js/base/components/button/index.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import React from 'react';
```

Comment:
We shouldn't need to import this.  With the current configuration I can build without this present without issues. So you can remove this import here and in other files you've added the import.

Reply:
Thanks for picking this up, was introduced when I was flailing around with a broken webpack build early on :) 88dddcfa88310adae4c644a429f2eeb68e4da73a